docs/source/challenge-doc.rst

```diff
@@ -365,6 +370,43 @@ List of states variables:
         - Ankle velocity
 
 
+Testing environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+To increase the accessibility of the task, two set of testing environment is provided for participants to familiarise themselves with the tasks. 
+Please note that the variation parameters are subject to change in the actual evaluation environment.
+
+The two environments are :code:`myoChallengeOslRunRandom-v0` and :code:`myoChallengeOslRunFixed-v0` and can be accessed via :code:`env = gym.make(“myoChallengeOslRunRandom-v0”, normalize_act=False)`
+
+The :code:`myoChallengeOslRunFixed-v0` environment is a simplified version of the :code:`myoChallengeOslRunRandom-v0` environment for participants to begin with, with a flat ground and a shorter 15m track. 
+
+The :code:`myoChallengeOslRunRandom-v0` includes a 30 meters track with a 20m stretch of flat ground, and the remaining 10m of increasingly difficult terrain
```

Comment:
only 10 m of difficult ground might be too short... can we make this longer? @elladyr  @P-Schumacher

Reply:
I am extending the track to be 100m, so there will be at least 80m of difficult terrain. Will update the documentation after the terrain has been correctly added.

Reply:
is this already in a commit?

Reply:
Not yet. I am still tuning the parameters for 100m. It will be included with the locoaction_fix PR. Should be done in another 15min

Reply:
Fixed in https://github.com/MyoHub/myosuite/pull/230/commits/14dbf56b1b1ab58e249eb5c11b5375ba0b2f9199